controller_interface/include/controller_interface/controller_interface_base.hpp

```diff
@@ -159,6 +158,23 @@ class ControllerInterfaceBase : public rclcpp_lifecycle::node_interfaces::Lifecy
   CONTROLLER_INTERFACE_PUBLIC
   const std::string & get_robot_description() const;
 
+  /**
+   * Method used by the controller_manager as a base NodeOptions to instantiate the Lifecycle node
+   * of the controller upon loading the controller.
+   *
+   * \note The controller_manager will add modify these NodeOptions in case, a params file is passed
+   * by the spawner to load the controller parameters or when controllers are loaded in simulation
```

Comment:
```suggestion
   * \note The controller_manager will modify these NodeOptions in case a params file is passed
   * by the spawner to load the controller parameters or when controllers are loaded in simulation
```